src/cursor/cur_table.c

```diff
@@ -22,6 +22,16 @@ static int __curtable_update(WT_CURSOR *cursor);
         }                                                                               \
     } while (0)
 
+#define APPLY_CG_CUSTOM(ctable, f)                                                      \
```

Comment:
I'm not really sure why these are macros - could they be made into functions? Is it because the function signature that is called differs and can't easily be passed as a parameter?

Reply:
I have removed this, as we don't need this anymore